src/main/scala/ai/datahunters/md/config/processing/AllowedExtensionsConfig.scala

```diff
@@ -0,0 +1,11 @@
+package ai.datahunters.md.config.processing
+
+case class AllowedExtensionsConfig(extensions: Option[Seq[String]] = None)
```

Comment:
@WojtekMin I thinkg we don't need this class. Especially when you added `extensions` field to `ProcessingConfig`. I woul remove it and use only `ProcessingConfig`